main.cpp

```diff
@@ -261,201 +298,206 @@ struct StereoWidenerAudioPlugin
-/*
+void StereoWidenerAudioPlugin::captureAudio(){}
+char StereoWidenerAudioPlugin::textInfo()
+    {
+    return 'g';
+    }
+float StereoWidenerAudioPlugin::widenSignal()
+    {
+    return 100.0f;
+    }
 
-Thing 5) Graphical User Interface
 
-*/
 struct GraphicalUserInterface
 {
-
-    //width in cm (int)
     int GUIWidth = 10;
-    //height in cm (int)
     int GUIHeight = 10;
-    //number of frequency selector dials (int)
     int freqDialsQuantity = 3;
-    //names of frequency selector dials (std::string)
     std::string freqDialsNames = ("Freq" "Gain" "Q");
-    //color of background (std::string)
     std::string backgroundColor = "Purple";
 
-    //3 things it can do:
-    // display input level
-    float displayInputLevel(); //return is displayed for user
-
-    // display attenuation level(s) of frequency band(s)
-    float displayAttenuation(); //return is displayed for user
-
-    // allow user to modify parameters
-    float parameterMod(); //return is related to function parameter
+    float displayInputLevel(); 
+    float displayAttenuation();
+    float parameterMod();
 };
-/*
 
-Thing 6) License
+float GraphicalUserInterface::displayInputLevel()
+    {
+    return 1.0f;
+    }
+float GraphicalUserInterface::displayAttenuation()
+    {
+    return 93.5f;
+    }
+float GraphicalUserInterface::parameterMod()
+    {
+    return 44.44f;
+    }
+
 
-*/
 struct License
 {
-
-    //text (std::string)
     std::string bodyText = "You must be this tall to ride this ride.";
-    //font size (int)
     int bodyTextFontSize = 18;
-    //format (std::string)
     std::string fileFormat = "PDF";
-    //text color (std::string)
     std::string bodyTextColor = "black";
-    //executed (bool)
     bool isExecuted = true;
 
-    //3 things it can do:
-    // provide information
-    char displayTextBody(); //return is displayed for user
-
-    // link to manufacturer website
-    char website(); //return is actionable by user
-
-    // not be copied
+    char displayTextBody();
+    char website();
     void copyProtection();
 };
-/*
 
-Thing 7) Company
+char License::displayTextBody()
+    {
+    return 't';
+    }
+char License::website()
+    {
+    return 'w';
+    }
+void License::copyProtection(){}
 
-*/
 struct Company
 {
-
-    //name (std::string)
     std::string companyName = "SBL Plugins";
-    //location (std::string)
     std::string companyLocation = "Atlanta";
-    //number of employees (int)
     int companyEmployees = 1;
-    //amount of income per year (float)
     float companyIncome = 0.0f;
-    //cost of office equipment (float)
     float equipmentCost = 200.25f;
 
-    //3 things it can do:
-    // create plugins
-    int createPlugin(); //return is related to function parameter
-
-    // set authorization methods
+    int createPlugin();
     void authEnable();
-
-    // sign legal agreements
     void signContract();
 };
-/*
 
-Thing 8) Signal Processor
+int Company::createPlugin()
+    {
+    return 1;
+    }
```

Comment:
Fix this formatting
Why would a function called 'createPlugin' return an integer, and specifically why would it return '1'?  The code needs to make sense for the action that the function name is implying.
- [ ] revise

Reply:
Understood. Changed/fixed.